### PR TITLE
L11 support

### PR DESCRIPTION
**Pull request by bmartus**
Looking to get Laravel 11 support for this package .. hoping this PR gets you most of the way there.

I'm not too familiar with the `test.yml` workflows portion, but I updated it to what I thought may work.  🤞

Thanks!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -11,15 +11,21 @@ jobs:
 
     strategy:
       matrix:
-        php: [ '7.4', '8.0', '8.1' ]
-        laravel: [ '8', '9', '10' ]
+        php: [ '7.4', '8.0', '8.1', '8.2' ]
+        laravel: [ '8', '9', '10', '11' ]
```
Comment (ptondereau) [inline .github/workflows/test.yml:15]: Is there anything compatible with PHP 8.3? We can test it against 8.3 I think

**Review by ptondereau — COMMENTED** [review]
Thanks for the PR! Just one nit-pick here

File: .github/workflows/test.yml (hunk 1 above)
Comment (bmartus) [reply .github/workflows/test.yml:15]: Probably no harm adding 8.3 -- I've pushed that change up.

**Comment by bmartus** [comment]
I updated some dependencies but the older versions of Laravel and PHP don't seem to be playing nice.  

Not sure if you want to drop support for L8, L9, 7.4, and php 8.0?

**Comment by ptondereau** [comment]
> I updated some dependencies but the older versions of Laravel and PHP don't seem to be playing nice.  
> 
> Not sure if you want to drop support for L8, L9, php 7.4 & 8.0?

Good catch. Yeah my rule of thumb here is just keep PHP versions still maintained.
So you can get rid of 7.4 8.0 with associated Laravel version

File: composer.json [hunk 2]
```diff
@@ -4,14 +4,14 @@
   "keywords": ["laravel", "framework", "UPS", "Laravel UPS Api", "Laravel-Ups-Api", "Pierre Tondereau", "Ptondereau"],
   "require": {
     "php": "^7.4.15 || ^8.0",
```
Comment (ptondereau) [inline composer.json:6]: This part need to be updated too

**Review by ptondereau — APPROVED** [review]
lgtm thank you

**Pull request merged**